packages/wrangler/how-to-bindings.md

```diff
@@ -0,0 +1,18 @@
+# How to register a binding
+
+1. Register wrangler.toml section in: `packages/wrangler/src/config/environment.ts` [ref](https://github.com/cloudflare/workers-sdk/blob/ce7db9d9cb4f5bcd5a326b86dde051cb54b999fb/packages/wrangler/src/config/environment.ts#L431-L451)
+1. Register validation functions in: `packages/wrangler/src/config/validation.ts` [ref-1](https://github.com/cloudflare/workers-sdk/blob/ce7db9d9cb4f5bcd5a326b86dde051cb54b999fb/packages/wrangler/src/config/validation.ts#L1297-L1306)
```

Comment:
This looks fine in a markdown viewer (the numbers are corrected), but looks odd in the raw state.

Reply:
That's a feature of markdown imo. You don't need to manually mess with bullet numbers, it will do it for you.